user_docs/en/changes.t2t

```diff
@@ -9,6 +9,7 @@ What's New in NVDA
 - New key commands:
   - New Quick Navigation command ``p`` for jumping to next/previous text paragraph in browse mode. (#15998, @mltony)
   - New unassigned Quick Navigation command to jump to the next/previous figure. (#10826)
+  - New unassigned Quick Navigation commands to jump to the next/previous menu item, toggle button, progress bar and math formula. (#16001, @mltony)
   -

```

Comment:
it might be worth combining these and creating a sublist

```suggestion
  - New unassigned Quick Navigation commands, which can be used to jump to the next/previous:
    - figure (#10826)
    - menu item (#16001, @mltony)
    - toggle button (#16001, @mltony)
    - progress bar (#16001, @mltony)
    - math formula (#16001, @mltony)
    -
  -
```